examples/typescript/package.json

```diff
@@ -12,7 +12,7 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-scripts": "^3.4.4",
-    "typescript": "^4.0.5"
+    "typescript": "^3.2.1"
```

Comment:
Needed because it's "react-scripts": "^3.4.4" peer dependency

Reply:
dependabot will bump this dependency if we downgrade it manually though